### ISSUE TITLE
feature suggestion: move version information, add help option

### ISSUE THREAD

**Issue by nbehrnd** [MehmetAzizYirik/MAYGEN#81]
The present executable of the application is distributed as `MAYGEN-1.7.jar`.  For future
releases of the application, I suggest the name stays fixed, i.e. `MAYGEN.jar` as used,
e.g. in the documentation.

As a result, it is easier to use the application with a moderating script, *regardless* of the
release of `MAYGEN` currently deployed.  Thus, if there are improvements of `MAYGEN`,
there is no need to open/edit/update these scripts moderating `MAYGEN`.

Similar to other applications (e.g. `ps2eps` in Linux Debian, or [date2name](https://github.com/novoid/date2name)\
moderated by Python), I suggest
- copy the usage information into a help menu on the CLI, accessed by `-h`, and/or `--help`
- offer access to the release information of the program by `-v`, and/or `--version`.

**Comment by MehmetAzizYirik** [comment]
@nbehrnd Thanks for the suggestion. Yes, when the MVN project is built, it adds also the version number to the file name like MAYGEN-1.7.jar. We will fix it to only MAYGEN in the future releases. But for the current version or the former releases, users can change the file name to whatever they prefer.

**Comment by javadev** [comment]
No, no. Version is important. I suggest to keep it. You may rename jar file after downloading.

**Comment by nbehrnd** [comment]
> No, no. Version is important. I suggest to keep it. You may rename jar file after downloading.
> 

Let me clarify:  The suggest is not about shipping MAYGEN without a
version information.  The suggest is about moving the information
about the release used into an output optionally accessible.

Norwid

**Comment by javadev** [comment]
@nbehrnd 

Good suggestion. Thank you for the issue.

**Comment by MehmetAzizYirik** [comment]
@nbehrnd Due to the common practice among java developers as Valentyn (@javadev ) recommended, we decided to keep the version in the name. We updated the readme and the jar file usage with the version number and added -h option for help in [V1.8](https://github.com/MehmetAzizYirik/MAYGEN/releases/tag/V1.8). Actually also without that, the jar file printed the usage information in V1.7 same as in V1.8.
